src/dao/PlatformEmitterDAO.js

```diff
@@ -1,7 +1,27 @@
 import AbstractContractDAO from './AbstractContractDAO'
+import TransactionModel from '../models/TransactionModel'
+import web3utils from 'web3/lib/utils/utils'
+import ContractsManagerDAO from '../dao/ContractsManagerDAO'
+import ERC20DAO from '../dao/ERC20DAO'
 
 export default class PlatformEmitterDAO extends AbstractContractDAO {
   constructor (at) {
     super(require('chronobank-smart-contracts/build/contracts/ChronoBankPlatformEmitter.json'), at)
   }
+
+  static async prepareTxsMap (txs): Promise<Map<TransactionModel>> {
```

Comment:
change to Promise<Immutable.Map<string,TransactionModel>>
where string is the type of map keys (but I don't sure whether this is the correct syntax – better check it)